test/components/evaluators/test_context_relevance_evaluator.py

```diff
@@ -235,10 +248,28 @@ def test_live_run(self):
         questions = ["Who created the Python language?"]
         contexts = [["Python, created by Guido van Rossum, is a high-level general-purpose programming language."]]
 
-        evaluator = ContextRelevanceEvaluator()
+        evaluator = ContextRelevanceEvaluator(progress_bar=False)
         result = evaluator.run(questions=questions, contexts=contexts)
 
         required_fields = {"individual_scores", "results", "score"}
         assert all(field in result for field in required_fields)
         nested_required_fields = {"score", "statement_scores", "statements"}
         assert all(field in result["results"][0] for field in nested_required_fields)
+
+    def test_all_statements_are_scored(self):
+        from haystack.components.evaluators import ContextRelevanceEvaluator
+
+        questions = ["Who created the Python language?"]
+        contexts = [
+            [
+                "Python, created by Guido van Rossum in the late 1980s, is a high-level general-purpose programming language. Its design philosophy emphasizes code readability, and its language constructs aim to help programmers write clear, logical code for both small and large-scale software projects.",
```

Comment:
How many statements can be extracted from this sentence? 2 or 3?

Reply:
2

Reply:
The `Java is a high...` context also should have 2 statements + the `Scale is a..` gives 5 statements

Reply:
I've reverted this, so that it's more deterministic - the LLM can say it's 4 or 5